app/overrides/decidim/blogs/posts/index/overriden_message.html.erb.deface

```diff
@@ -0,0 +1,13 @@
+<!-- insert_top 'div.row.column' -->
+
+<% if paginate_posts.empty? %>
+    <div class="callout warning">
+        <%= t ".no_post_warning" %>
+    </div>
+<% end %>
+<% if current_participatory_space&.author == current_user && current_participatory_space&.state == "validating" %>
```

Comment:
I think not every participatory spaces responds to `:author`, it occurs a crash on processes blogs